docs/chapters/sdk/template-service.rst

```diff
@@ -0,0 +1,19 @@
+Template Service
+================
+
+A template service serves as an example CMake_ project for service developers that should be used as a starting point while creating platform services. To acquire the template service source code the user needs to fetch the code from github.
```

Comment:
I'd like it to be more than just a CMake example, it should at least show you the repository structure, having a propper readme so people have one in their repos too, a license file, and so on. And then I'd almast want those parts to be described in text here in the SWF so someone reading it could find all this information even if they didn't quite know that they should search on how do standardize for example the directory structure.